# HG changeset patch
# User Rik <rik@octave.org>
# Date 1501128559 25200
#      Wed Jul 26 21:09:19 2017 -0700
# Node ID 755b4c622972d2e16980bc433af016592b4df9e8
# Parent  4f12819a634fa25971c155656ade2363c1e9b9ab
Fix "variable may be used uninitialized" warning reported by gcc.
* pr-output.cc (octave_print_internal): Initialize fw to 0.

diff --git a/libinterp/corefcn/pr-output.cc b/libinterp/corefcn/pr-output.cc
--- a/libinterp/corefcn/pr-output.cc
+++ b/libinterp/corefcn/pr-output.cc
@@ -1738,17 +1738,17 @@ octave_print_internal (std::ostream& os,
             }
 
           if (i < nr - 1)
             os << "\n";
         }
     }
   else
     {
-      int fw;
+      int fw = 0;
       double scale = 1.0;
       set_format (m, fw, scale);
       int column_width = fw + 2;
       octave_idx_type total_width = nc * column_width;
       octave_idx_type max_width = octave::command_editor::terminal_cols ();
 
       if (pr_as_read_syntax)
         max_width -= 4;
